### PR TITLE
Removed username and password from the options menu

### PR DESCRIPTION
**Pull request by jeremy-shannon**
Plus a few minor style things related to the loading screen, if you're interested.

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -164,7 +164,11 @@ chrome.runtime.onMessage.addListener((request) => {
         hideLoading();
       });
   } else if (request.prsUpdated) {
-    updatePRs();
+    updatePRs()
+      .then(()=>{
+        hideLoading();
+      });
```
Comment (gbockus) [inline src/popup.js]: I don't think this then is necessary b/c the hideLoading function is called inside the updatePRs function.

File: src/popup.js [hunk 2]
```diff
@@ -164,7 +164,11 @@ chrome.runtime.onMessage.addListener((request) => {
         hideLoading();
       });
   } else if (request.prsUpdated) {
-    updatePRs();
+    updatePRs()
+      .then(()=>{
```
Comment (gbockus) [inline src/popup.js]: Just to give you a taste of the fully nit picking I'd usually do for the fat arrows you should have a space on either side for readability. 

```
     .then(() => { 
```

**Comment by gbockus** [comment]
good stuff...a small nit on the loading image change.  If you could either tell me I'm full of it or push another commit to remove the call that isn't necessary I'll merge this puppy.

**Comment by jeremy-shannon** [comment]
You kiddin' me?! Nitpick away! How else am I supposed to learn? :)

Honestly, I'm not even sure how that snuck in there. Maybe it was some other change that I rolled back and just forgot this? No idea. I'll fix it as soon as I get a chance.

**Comment by jeremy-shannon** [comment]
So I made the corrective commit and I pushed to up to the repo, but it looks like it automatically pushed the commit up here as well. Is that normal behavior on Github? Is it because I'm now a contributor that my commits to my forked repo are automatically added to my PR in this repo? :)

**Comment by gbockus** [comment]
whoa...dammit I need a monitor or something....Yeah if you have an open PR from your fork to to the original repo and you push a new commit onto the brach that your have the PR open against it will show up here as well.  You are merging the branch at the time you push the button.  If you want to avoid that behavior  your should use multiple branches.

**Pull request merged**